src/include/migraphx/value.hpp

```diff
@@ -140,9 +140,9 @@ struct value
     using const_pointer   = const value_type*;
     using array           = std::vector<value>;
     using object          = std::unordered_map<std::string, value>;
-    struct binary : std::vector<std::uint8_t>
+    struct binary : std::vector<std::int8_t>
```

Comment:
Is there a reason you can't cast it to `char*` like what we do for [msgpack](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/src/msgpack.cpp#L88)?

This also breaks the unit tests.

Reply:
No problem, I tried this first but didn't work so I added the cast and forgot to remove this.